nifi-extension-bundles/nifi-update-attribute-bundle/nifi-update-attribute-ui/src/main/webapp/js/application.js

```diff
@@ -1484,14 +1487,33 @@ var ua = {
                 conditions.push(condition.expression);
             });
             return conditions;
-        } else {
-            var actions = [];
-            $.each(rule.actions, function (_, action) {
-                actions.push(action.attribute);
-                actions.push(action.value);
-            });
-            return actions;
-        }
+        } else if (filterType.value === 'action') {
+                   var actions = [];
```

Comment:
The indentation appears to be off in this new block, in comparison with the other blocks.

Reply:
Thank you I didn't notice that I thought my IDE auto formatted it.